modules/core/pom.xml

```diff
@@ -83,6 +83,14 @@
       <artifactId>zookeeper</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.hdrhistogram</groupId>
```

Comment:
what are the license of these?  Does not matter now... but if we ever try to incubate at Apache then Fluo can only depend on certain licenses.